projects/1.20/assets/slashblade/slashblade/lang/zh_cn.json

```
@@ -0,0 +1,177 @@
+  "adv.slashblade.judgement_cut_just": "剑技 -裂空审判 EX-     ",
+  "adv.slashblade.judgement_cut_just.desc": "使用按键\n攻击 ♠:%s\n♠ 按住,蓄力完毕时放开",
+  "adv.slashblade.quick_charge": "剑技 -迅速蓄力-         ",
+  "adv.slashblade.quick_charge.desc": "使用按键\n攻击 ♠:%s\n♠ 在纳刀的瞬间按下攻击 ♠",
+
+  "adv.slashblade.rapid_slash": "剑技 -迅冲斩-         ",
+  "adv.slashblade.rapid_slash.desc": "使用按键s\n攻击 ♠:%s\n前进 ᐃ:%s\n潜行 ⚪:%s\n在地上\n⚪+ᐃ+♠",
+  "adv.slashblade.rising_star": "剑技 -星旋天翔-         ",
+  "adv.slashblade.rising_star.desc": "使用按键s\n攻击 ♠:%s\n迅冲斩之后\n♠ 按住直到命中敌人",
+
+  "adv.slashblade.upperslash": "剑技 -上斩-         ",
+  "adv.slashblade.upperslash.desc": "使用按键s\n攻击 ♠:%s\n后退 ᐁ:%s\n潜行 ⚪:%s\n在地上\n⚪+ᐁ+♠",
+  "adv.slashblade.upperslash_jump": "剑技 -跃升斩-         ",
+  "adv.slashblade.upperslash_jump.desc": "使用按键s\n攻击 ♠:%s\n上斩后\n♠ 按住",
+
+
+  "adv.slashblade.air_trick":   "能力 -隔空瞬步-            ",
+  "adv.slashblade.air_trick.desc": "向注视中的敌人的头上瞬间移动\n使用按键\n走行 ♠:%s\n前进 ᐃ:%s\n潜行 ⚪:%s\n⚪+ᐃ+♠",
+  "adv.slashblade.trick_down":  "能力 -瞬步退行-           ",
+  "adv.slashblade.trick_down.desc": "在空中迅速落地\n使用按键\n走行 ♠:%s\n后退 ᐁ:%s\n潜行 ⚪:%s\n⚪+ᐁ+♠",
+  "adv.slashblade.trick_dodge": "能力 -瞬步回避-          ",
+  "adv.slashblade.trick_dodge.desc": "往任意方向瞬间移动躲避伤害\n使用按键\n走行 ♠:%s\n在移动中 ♠",
+  "adv.slashblade.trick_up":    "能力 -瞬步上行-             ",
+  "adv.slashblade.trick_up.desc": "向上瞬间移动\n使用按键\n走行 ♠:%s\n前进 ᐃ:%s\n潜行 ⚪:%s\n⚪+ᐃ+♠",
+  "adv.slashblade.enemy_step":  "能力 -借力跳-           ",
+  "adv.slashblade.enemy_step.desc": "在空中把敌人作为垫脚石以二段跳\n使用按键\n跳跃 □:%s\n在空中靠近敌人时 □",
+  "adv.slashblade.kick_jump":   "能力 -踢跃-           ",
+  "adv.slashblade.kick_jump.desc": "蹬开墙壁再度跳跃\n使用按键\n跳跃 □:%s\n在空中靠近墙壁时 □",
+
+  "adv.slashblade.guard":       "能力 -防御-                ",
+  "adv.slashblade.guard.desc": "消耗耐久度将敌人的远程攻击无效\n'行走'中无法发动\n使用按键\n潜行 ⚪:%s\n⚪",
+  "adv.slashblade.guard_just":  "能力 -格挡-           ",
+  "adv.slashblade.guard_just.desc": "将敌人的攻击无效\n高评价状态中不会消耗耐久度\n评价会上升\n使用按键\n潜行 ⚪:%s\n在受到攻击时 ⚪",
+
+  "adv.slashblade.summonedswords": "射技 -幻影剑-  ",
+  "adv.slashblade.summonedswords.desc": "消耗经验值射出幻影剑\n使用按键\n%s ♠:%s",
+  "adv.slashblade.spiral_swords":  "射技 -环形幻剑阵-    ",
+  "adv.slashblade.spiral_swords.desc": "消耗5经验值在自身周围形成幻影剑圆阵。\n使用按键\n%s ♠:%s\n♠ 按住",
+  "adv.slashblade.storm_swords":   "射技 -怒风幻剑阵-     ",
+  "adv.slashblade.storm_swords.desc": "消耗5经验值将幻影剑如同怒风般包围攻击并击飞敌人。\n使用按键\n%s ♠:%s\n后退 ᐁ:%s\n潜行 ⚪:%s\n⚪+ᐁ+♠ 按住",
+  "adv.slashblade.blistering_swords": "射技 -急袭幻剑阵-",
+  "adv.slashblade.blistering_swords.desc": "消耗5经验值将幻影剑展开快速攻击敌人\n使用按键\n%s ♠:%s\n前进 ᐃ:%s\n潜行 ⚪:%s\n⚪+ᐃ+♠ 按住 放开",
+  "adv.slashblade.heavy_rain_swords": "射技 -暴雨幻剑阵-",
+  "adv.slashblade.heavy_rain_swords.desc": "消耗5经验值将幻影剑如同五月雨般在敌人头上降下。\n使用按键\n%s ♠:%s\n后退 ᐁ:%s\n前进 ᐃ:%s\n潜行 ⚪:%s\n⚪+ᐁ to ᐃ+♠ 按住",
+
+
+
+  "adv.slashblade.exeffect": "附魔效果",
+  "adv.slashblade.exeffect.desc": "附魔提供额外的效果提升",
+
+  "adv.slashblade.exeffect.soul_speed": "灵魂疾行",
+  "adv.slashblade.exeffect.soul_speed.desc": "格挡的指令输入时间会随着灵魂疾行的等级提升延长1tick。",

```

```suggestion
  "adv.slashblade.exeffect.soul_speed.desc": "格挡指令的输入时间会随着灵魂疾行的等级提升，每1级延长1tick。",
```
原文`ジャスト入力の受付時間がSoulSpeedのレベル1毎に1tick延長されます`
感觉有点怪